### PR TITLE
resin-init-board: modprobe i2c-dev on startup

### PR DESCRIPTION
**Pull request by michal-mazurek**
connect https://github.com/resin-os/resinos/issues/201

### CODE REVIEW

**Comment by floion** [comment]
Changelog entry please?

**Comment by floion** [comment]
ping @michal-mazurek ?

File: layers/meta-resin-raspberrypi/recipes-support/resin-init/files/resin-init-board [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+#
```
Comment (agherzan) [inline layers/meta-resin-raspberrypi/recipes-support/resin-init/files/resin-init-board]: any reason for this line here?

Comment (michal-mazurek) [inline layers/meta-resin-raspberrypi/recipes-support/resin-init/files/resin-init-board]: It seems to be customary in scripts. On the third line there would usually be a description or licence. In this case there is no comment, but I felt this looks better for consistency. I can remove it if it's ugly.

Comment (agherzan) [inline layers/meta-resin-raspberrypi/recipes-support/resin-init/files/resin-init-board]: If we have no license here then we can remove it completely.

**Pull request merged**